integration-tests/utils/aws/cloudWatchGetLogs.ts

```diff
@@ -1,141 +1,116 @@
-import { INITIATE_DATA_REQUEST_LAMBDA_LOG_GROUP } from '../../constants/awsParameters'
 import { cloudWatchLogsClient } from './cloudWatchLogsClient'
 import {
-  DescribeLogStreamsCommandInput,
-  DescribeLogStreamsCommandOutput,
   DescribeLogStreamsCommand,
   LogStream,
-  FilterLogEventsCommandInput,
   FilterLogEventsCommand,
-  FilterLogEventsCommandOutput,
   FilteredLogEvent
 } from '@aws-sdk/client-cloudwatch-logs'
 
-export const getLogStreamPrefix = () => {
```

Comment:
Can you please leave the old functions there temporarily, as my local branch needs them? I'll refactor to use them if possible, after I merge. The athena sql events are a bit different from the initiate ones.